### PR TITLE
Refs [TASK][34039] Create UI List Meal Screen

### PR DESCRIPTION
**Pull request by phucsaker99**
## Related Tickets
- [#Task 34039: Create UI List Meal Screen](https://edu-redmine.sun-asterisk.vn/issues/34039)
## WHAT
- Create RecylerView list meals (by category, areas or ingredients)
- Create TextView Title
## Evidence (Screenshot or Video)
image
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_meal_list.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        android:id="@+id/view"
+        android:layout_width="0dp"
+        android:layout_height="?attr/actionBarSize"
+        android:background="@color/red_berry"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
+        android:id="@+id/view2"
```
Comment (thonc-2135) [inline app/src/main/res/layout/activity_meal_list.xml]: ghi rõ view gì ra nhé

Comment (phucsaker99) [reply app/src/main/res/layout/activity_meal_list.xml]: vâng a

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -25,4 +26,5 @@
     <dimen name="dp_200">200dp</dimen>
     <dimen name="dp_300">300dp</dimen>
     <dimen name="dp_400">400dp</dimen>
+    <dimen name="dp_18">18dp</dimen>
```
Comment (huytq-2422) [inline app/src/main/res/values/dimens.xml]: sắp xếp

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -39,19 +40,21 @@
...
-            android:autofillHints=""
             android:background="@drawable/custom_edit_text"
-            android:drawableStart="@drawable/ic_search"
+            android:drawablePadding="@dimen/dp_15"
+            android:gravity="center_vertical"
             android:hint="@string/text_search_meal"
-            android:inputType="text"
-            android:padding="@dimen/dp_5"
+            android:paddingStart="@dimen/dp_10"
+            android:paddingEnd="@dimen/dp_10"
```
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_home.xml:55]: paddingHorizontal

File: app/src/main/res/layout/activity_meal_list.xml [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        android:text="@string/text_back"
+        android:textColor="@android:color/white"
+        android:textSize="@dimen/sp_20"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="@+id/buttonBack"
+        app:layout_constraintStart_toEndOf="@+id/buttonBack"
+        app:layout_constraintTop_toTopOf="@+id/buttonBack" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerListMeal"
```
Comment (huytq-2422) [inline app/src/main/res/layout/activity_meal_list.xml]:
```suggestion
        android:id="@+id/recyclerMeals"
```

**Pull request merged**